src/Illuminate/Auth/Console/AuthMakeCommand.php

```diff
@@ -63,6 +64,28 @@ public function handle()
             );
         }
 
+        if (! $this->option('no-tests')) {
```

Comment:
I thought about it, but I am unsure if this really simplifies things... Let's see what others think.